src/main/java/core/basesyntax/model/Activity.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.model;
+
+public class Activity {
```

Comment:
I don't know if this is a mistake, but you should rename this class to e.g. TrancastionDto

Reply:
This is not DAO it is entities of our program

Reply:
I'm not talking about Dao, read what DTO is.